### PR TITLE
update resources documentation to address INTMDB-225

### PR DESCRIPTION
**Pull request by nikhil-mongo**
## Description

update resources documentation to address INTMDB-225 for including atlas_cidr_block in the examples and also used updated syntax for interpolation in the examples.

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

File: website/docs/r/network_peering.html.markdown
Comment (bartpolot) [inline website/docs/r/network_peering.html.markdown]:
```suggestion
  atlas_cidr_block = local.ATLAS_CIDR_BLOCK
```

File: website/docs/r/network_peering.html.markdown
Comment (bartpolot) [inline website/docs/r/network_peering.html.markdown]:
```suggestion
  atlas_cidr_block      = mongodbatlas_network_container.test.atlas_cidr_block
```

**Review by bartpolot — CHANGES_REQUESTED** [review]
This minimizes the chance of a user getting an error, in case the CIDR block in the config file doesn't match with the hardcoded string in the file. Plus it avoids hardcoded strings :)

File: website/docs/r/network_peering.html.markdown
Comment (nikhil-mongo) [reply website/docs/r/network_peering.html.markdown]: updated

File: website/docs/r/network_peering.html.markdown
Comment (nikhil-mongo) [reply website/docs/r/network_peering.html.markdown]: updated

**Review by themantissa — APPROVED** [review]
LGTM, thank you @nikhil-mongo !!

**Comment by themantissa** [comment]
@bartpolot will wait on your final approval.  TY!

**Review by bartpolot — APPROVED** [review]
LGTM, Thanks! :)

**Pull request merged**
